book/html.md

```diff
@@ -53,23 +50,19 @@ class Element:
         self.parent = parent
 ```
 
-I added a `children` field to both `Text` and `Element`, even though
```

Comment:
Why did you remove this and the footnote above?

Reply:
Uhh, I meant to remove exactly one.